### DISCUSSION TITLE
Need to know what OS versions(Android and iOS), browser versions and what kind of browser is used.

### DISCUSSION THREAD

**Discussion by Bharathkrently** [responsively-org/responsively-app#679] [Q&A] [answered]
Can someone let me know what OS versions(Android and iOS), browser versions, and what kind of browser(Chrome/Safari/Firefox etc) is used in Responsively.app.

**Comment by jjavierdguezas** [comment, marked as answer]
Responsively uses electron ^9.3.1 so it use chromium version ~[83](https://www.electronjs.org/blog/electron-9-0) (83.0.4103.122 [according to this package](https://github.com/Kilian/electron-to-chromium/blob/9eaf581b7594cf70d41e36b29ba9c66441518e77/full-chromium-versions.js#L848))
Responsively do not emulate OS, it just change the user agent, screen size, etc (you can config that via custom devices)

**Comment by gennahumpal** [comment]
Msi labtop linked to note 20 ultra phone while doing a 07 chevy truck diagnostic??
